master/docs/manual/cfg-www.rst

```diff
@@ -119,10 +119,10 @@ The available classes are described here:
 
     ::
 
-        from buildbot.plugins import util
+        from buildbot.www import auth
         c['www'] = {
             # ...
-            'auth': util.UserPasswordAuth({"homer": "doh!"}),
+            'auth': auth.UserPasswordAuth({"homer": "doh!"}),
         }
 
 .. py:class:: buildbot.www.auth.HTPasswdAuth(passwdFile)
```

Comment:
After a bit of thinking (sorry), I believe the best will be to put it in a separate namespace: `buildbot.plugins.utils.www.HttPasswdAuth` etc...